app/services/usps_in_person_proofing/enrollment_helper.rb

```diff
@@ -98,6 +98,21 @@ def handle_standard_error(err, enrollment)
       def analytics(user: AnonymousUser.new)
         Analytics.new(user: user, request: nil, session: {}, sp: nil)
       end
+
+      def transliterate(value)
+        return value unless IdentityConfig.store.usps_ipp_transliteration_enabled
+
+        result = transliterator.transliterate(value)
+        if result.unsupported_chars.size > 0
+          result.original
+        else
+          result.transliterated
+        end
+      end
+
+      def transliterator
+        Transliterator.new
```

Comment:
this is stateless, so we could micro-optimize and save a few allocations by memoizing

or, we could probably just make `transliterate` a static method?

```suggestion
        @transliterator ||= Transliterator.new
```

Reply:
This is exactly what I tried before, and it caused the mock to be preserved between tests (and therefore the tests failed). I also don't think this micro-optimization has a high payoff.

Reply:
oh I just saw that there's a `class << self` around all of this, no wonder there was test pollution. It's probably time to maybe split this class up and maybe maybe it stateful (ex `InPersonEnroller.new(user).create_enrollment` vs generic `Helper`), but that's definitely outside the scope of this PR